spec/models/food_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  subject do
+    User.create(name: 'Miliyon', email: 'miliayalew@gmail.com', password: '123456')
+  end
+
+  before(:each) do
+    @food = Food.new(name: 'Burger', measurement_unit: 'gram', price: 230, quantity: 4, user_id: subject.id)
+  end
+
+  context 'Testing validations' do
+    it 'Validation should be successful' do
+      expect(@food).to be_valid
+    end
+
+    it 'Food should have a name' do
+      expect(@food.name).to be_present
+    end
+
```

Comment:
- [ ] Good job adding both unit and integration tests. Unfortunately, some of your test cases are failing as seen in the screenshot below. Kindly fix these failing tests.
![image](https://user-images.githubusercontent.com/62849872/217958549-b99e0f7b-21c2-40d9-945b-e50a4336ccdd.png)

Reply:
Hi @t-yanick, those tests are passing on both, my computer and my partner's computer. We have tried everything and it is working. I will do a video to validate that the testing is correct.

Reply:
https://user-images.githubusercontent.com/105117832/217973918-9901cd88-f985-4df7-83a5-9a834cf8ef60.mp4

Reply:
- [ ] Hi Team, great job. You can request for a re-review and the next reviewer will handle it.

Reply:
Hey @t-yanick , Thanks for the review and feedabck